tasks/tasks.js

```diff
@@ -33,49 +32,18 @@ const weeklyLeaderboardResults = (client) => new CronJob('0 11 * * 1', async fun
     }
 });
 
-// RSVP day before meeting message
-// const dayBeforeReminder = (client) => new CronJob('1 12 * * 1', function() {
-//     const row = new MessageActionRow()
-//         .addComponents(
-//             new MessageButton()
-//                 .setCustomId('add')
-//                 .setLabel('Add')
-//                 .setStyle('SUCCESS'),
-//             new MessageButton()
-//                 .setCustomId('remove')
-//                 .setLabel('Remove')
-//                 .setStyle('DANGER'),
-//         );
-
-//     const dayBeforeMsg = new MessageEmbed()
-//         .setColor('#0080ff')
-//         .setTitle('Club meeting this Tuesday at 6pm')
-//         .setDescription('Click buttons below to add / remove yourself from RSVP list** **')
-//         .setImage(
-//             'https://www.waketech.edu/themes/custom/talon/assets/images/wake-tech-2017.png',
-//         );
-
-//     client.channels.cache.get(targetChannel,
-//     ).send({ embeds: [dayBeforeMsg], components: [row] });
-
-// });
-
-// Meeting start reminder
-// const meetingStart = (client) => new CronJob('58 17 * * 2', function() {
-//     const mtgStartMsg = new MessageEmbed()
-//         .setColor('#0080ff')
-//         .setTitle('Club meeting starting now')
-//         .setDescription('Join General Voice Chat')
-//         .setImage(
-//             'https://www.waketech.edu/themes/custom/talon/assets/images/wake-tech-2017.png',
-//         );
-
-//     client.channels.cache.get(targetChannel,
-//     ).send({ embeds: [mtgStartMsg] });
-// });
+const resetLeaderboard = (client) => new CronJob('0 11 20 4,11 *', async function() {
+    try {
+        Users.destroy({ truncate : true });
```

Comment:
I think either one works. As soon a user is awarded points again, the table is created. It's something I can look into though. It might be better to just reset like you suggested. Thank you for the feedback.